types/express-enforces-ssl/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for express-enforces-ssl 1.1
+// Project: https://github.com/aredo/express-enforces-ssl
+// Definitions by: Kevin Stubbs <https://github.com/kevinstubbs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Request, Response, NextFunction } from 'express';
+
+/**
+ * Enforces HTTPS connections on any incoming requests.
+ */
+declare function enforceHTTPS(): (req: Request, res: Response, next: NextFunction) => void;
+
+declare namespace enforceHTTPS {
```

Comment:
You don't need this namespace &mdash; it just allows people to incorrectly use `import * as expressEnforceSsl from 'express-enforce-ssl'`, instead of the correct `import expressEnforceSsl = require('express-enforce-ssl')` that you have in this test.